x/incentives/keeper/grpc_query_test.go

```diff
@@ -108,6 +108,39 @@ func (suite *KeeperTestSuite) TestGRPCActiveGauges() {
 	suite.Require().Equal(res.Data[0].String(), expectedGauge.String())
 }
 
+func (suite *KeeperTestSuite) TestGRPCActiveGaugesPerDenom() {
+	suite.SetupTest()
+
+		// initial check
+	res, err := suite.app.IncentivesKeeper.ActiveGaugesPerDenom(sdk.WrapSDKContext(suite.ctx), &types.ActiveGaugesPerDenomRequest{})
```

Comment:
oh you have to make a querier struct now for this to work now. (An update happened on main. 

Change to `suite.querier,ActiveGaugesPerDenom` and I think that should work in all places where this is erroring.